src/app/modules/consultations/consultation-profile/read-respond/read-respond.component.html

```diff
@@ -206,130 +206,202 @@
+            <div class="footer-space__left" *ngIf="responseSubmitted">
+              <div class="points-text">{{'You get ' + earnedPoints + ' points for submitting this!'}}</div>
+              <div class="link">{{'View Leaderboard' | translate}}</div>
+            </div>
+            <div class="footer-space__right">
+              <button class="app-btn secondary mt-xs" *ngIf="responseSubmitted" (click)="closeFeedbackModal()">
+                <p class="m-0">
+                  <span class="submit-logo">
+                    <picture class="citizen-logo">
+                      <source srcset="assets/images/Vector.webp" type="image/webp">
+                      <source srcset="assets/images/Vector.jpg" type="image/jpeg">
+                      <img src="assets/images/Vector.jpg" />
+                    </picture>
+                  </span>
+                  {{'Ok Great' | translate}}
+                </p>
+                <i class="fa fa-angle-right"></i>
+              </button>
+              <span class="mr-2" *ngIf="responseSubmitLoading">
+                <img src="assets/images/loading.svg">
+              </span>
+              <button class="app-btn secondary" *ngIf="!responseSubmitted" (click)="createResponse()">
+                <p class="m-0">
+                  <span class="submit-logo">
+                    <picture class="citizen-logo">
+                      <source srcset="assets/images/Vector.webp" type="image/webp">
+                      <source srcset="assets/images/Vector.jpg" type="image/jpeg">
+                      <img src="assets/images/Vector.jpg" />
+                    </picture>
+                  </span>
+                  {{'Select and submit' | translate}}
+                </p>
+                <i class="fa fa-angle-right"></i>
+              </button>
+            </div>
           </div>
         </div>
       </div>
     </div>
   </div>
+</div>
 
-  <app-confirm-email-modal *ngIf="showConfirmEmailModal" (close)="showConfirmEmailModal = false"></app-confirm-email-modal>
+<app-confirm-email-modal *ngIf="showConfirmEmailModal" (close)="showConfirmEmailModal = false">
+</app-confirm-email-modal>
+
+
+<!-- Thanks modal  -->
+<div class="thankyou-modal">
+  <div bsModal #thankyouModal="bs-modal" class="modal fade" tabindex="-1"
+    [config]="{show: false, backdrop: 'static', ignoreBackdropClick: true, keyboard: false}" role="dialog"
+    aria-labelledby="thankyouModal">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <div class="modal-body">
+          <div class="thankyou-inner">
+            <button class="close-btn" (click)="thankyouModal.hide()"><img src="assets/images/close.png" /></button>
+            <div class="thankyou-inner__top">
+              <img class="thankyou-img" src="assets/images/thankyou.png" />
+              <div class="thankyou-content">
+                <p>Thank you for your response</p>
+                <p>
```

Comment:
@Nishanthh 
- Can you make this text dynamic now? The field is there in the API on Staging. The existing text can be the default one in case the API field returns null for some reason.
- Also for private Consultations, the text is different and will be static. Check ticket description for more details
- For private Consultation, the share buttons should be visible. Check the ticket description for more details

Reply:
@Nishanthh Is this done?

Reply:
No No bro, doing that.

Reply:
Bro, what is the api field name for the dynamic message?